Library/ViewModels/PledgeViewUseCase.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+import KsApi
+import ReactiveSwift
+
+public protocol PledgeViewUseCaseInputs {
+  func goToPledgeViewTapped()
+}
+
+public protocol PledgeViewUseCaseOutputs {
+  var goToNativePledgeView: Signal<ManagePledgeViewParamConfigData, Never> { get }
+  var goToPledgeManagementViewPledge: Signal<URL, Never> { get }
```

Comment:
nit: rename to `goToPledgeManagementPledgeView` to align with `goToNativePledgeView`